interpreter/cling/test/lit.cfg

```diff
@@ -113,6 +112,11 @@ if platform.system() not in ['Windows'] or lit_config.getBashPath() != '':
 if platform.system() not in ['Windows']:
     config.available_features.add('not_system-windows')
 
+lit.util.usePlatformSdkOnDarwin(config, lit_config)
+macOSSDKVersion = lit.util.findPlatformSdkVersionOnMacOS(config, lit_config)
+if macOSSDKVersion is not None:
+    config.available_features.add('macos-sdk-' + str(macOSSDKVersion))
```

Comment:
This should not be needed, we don't use the `macos-sdk-*` features. The important call is probably `usePlatformSdkOnDarwin` which sets `SDKROOT` in the environment.

Reply:
Agreed. However, we tend to copy parts of the clang lit.cfg and they have it. That would enable us to do XFAIL on it for example and reduce the amount of differences we have against that file..

Reply:
Why? It's a separate file, I don't understand which "differences"? We don't need it right now, and I just removed a bunch of unused lines that were leading to warnings in debbd9bb1433400bb58ae19bca7bff4b61e2a83b...